docs/advanced-usage/compute-backends/slurm/index.md

```diff
@@ -6,30 +6,147 @@ backend Kubernetes and HTCondor.
 
 ## Authentication
 
-In order to use CERN Slurm cluster you need to be authenticated using
-Kerberos. [Generate keytab file](../../access-control/kerberos/index.md#generating-keytab-file)
+In order to use CERN Slurm cluster you need an access to Slurm Linux HPC resources.
+Please find more information about how to get access to the cluster in the official [CERN Batch Service documentation](https://batchdocs.web.cern.ch/linuxhpc/access.html).
+You also have to be authenticated using Kerberos. In order to do it please [generate keytab file](../../access-control/kerberos/index.md#generating-keytab-file)
 and [upload it and your CERN username as secrets to REANA](../../access-control/kerberos/index.md#uploading-secrets).
 
 ## Specifying compute backend
 
 In order to execute certain steps of a workflow on the CERN Slurm cluster
-you must specify ``slurmcern`` as the step's execution backend in the
-workflow specification.
-
-```yaml hl_lines="6"
-
-   # Serial example
-   ...
-   steps:
-      - name: reana_demo_helloworld_htcondorcern
-        environment: 'python:2.7-slim'
-        compute_backend: slurmcern
-        commands:
-            - python "${helloworld}"
+you must specify ``slurmcern`` as the step's ``compute_backend`` in the
+workflow specification. Please find [some examples](.#examples) how to do it in the bottom
+of the page.
+
+## Specifying environment
+
+There are several different ways to specify environments in the workflow
+specification for Slurm compute backend:
+
+- default way would be to specify a docker image which will be automatically converted to Singularity container
+- it is also possible to specify a Singularity image. Please note that the container
+file needs to be included into `reana.yaml` inputs and uploaded to the workspace before
+starting the workflow
+- you can also use Singularity from CVMFS unpacked images
```

Comment:
The Slurm jobs will run in a containerised compute environment that uses Singularity container technology. There are three possibilities how you can specify the desired computing environment for your job:

- If you are using a Docker container image in your workflow step, REANA will automatically convert the image into a Singularity SIF image before submitting the job. This is fully transparent.

- You can also specify your own Singularity SIF image. You have to upload it into your workspace before starting the workflow.

- You can also use Singularity images from the [CVMFS unpacked images](https://gitlab.cern.ch/unpacked/sync/-/blob/master/recipe.yaml) area.

Please see the [Examples](.#examples) section below that will provide concrete examples for each of these techniques.